ontimize/components/form/o-form.component.ts

```diff
@@ -768,6 +795,11 @@ export class OFormComponent implements OnInit, OnDestroy, CanComponentDeactivate
     this.formNavigation.goInsertMode(options);
   }
 
+  _clearFormAfterInsert() {
+    this.clearData();
+    this._setComponentsEditable(true);
+  }
```

Comment:
We still have to emit the onFormModeChange event here due to something can be subscribed to it in order to perform some form fiel initialization